websites/G/GetPocket/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "RedBird16.com",
+    "id": "444872133504663552"
+  },
+  "service": "GetPocket",
+  "description": {
+    "en": "When you find something you want to view later, put it in Pocket."
+  },
+  "url": [
+    "www.getpocket.com",
+    "getpocket.com",
+    "app.getpocket.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/7Qre9TD.png",
+  "thumbnail": "https://i.imgur.com/ac3w7ln.png",
+  "color": "#12171C",
```

Comment:
thumbnail should be a promotional banner or a screenshot of the website homepage

Reply:
ProtonMail? Why? It's not ProtonMail. It's Get Pocket.